packages/rocketchat-ui-flextab/flex-tab/tabs/membersList.coffee

```diff
@@ -64,6 +65,9 @@ Template.membersList.helpers
 		return '' unless roomData
 		return roomData.t in ['p', 'c'] and RocketChat.authz.hasAllPermission('add-user-to-room', this._id)
 
+	showUtcOffset: ->
+		return this.utcOffsetNum isnt Meteor.user().utcOffset
```

Comment:
It will.

My suggestion is
Get the user's utcOffset on line 22, before the loop and change the utcOffset if to pass a empty string if the values are the same. So we not even need an if